### PR TITLE
Icon button

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: frontend/src/pages/Home.js [hunk 1]
```diff
@@ -1,11 +1,50 @@
-import React from 'react';
+import * as React from 'react';
```
Comment (FortizFil) [inline frontend/src/pages/Home.js:1]: What is it for?

**Review by FortizFil — CHANGES_REQUESTED** [review]

File: frontend/src/components/ui-kit/components/IconButton/IconButton.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './IconButton.module.scss';
+
+const IconButton = (props) => {
+  return (
+    <img
```
[the remark this answers is not in the document]
Comment (Svitlana-G) [reply frontend/src/components/ui-kit/components/IconButton/IconButton.jsx]: Changed to button

File: frontend/src/components/ui-kit/components/IconButton/IconButton.module.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+.icon-border-yellow {
+  border-radius: 8px;
+  border: 1px solid #e1c429;
+  padding: 6px;
+
+  &:hover {
+    border: 1px solid $page-accent-color;
+    background-color: #fffdf2;
+    cursor: pointer;
```
Comment (FortizFil) [inline frontend/src/components/ui-kit/components/IconButton/IconButton.module.scss:16]: Remove this

File: frontend/src/components/ui-kit/components/IconButton/IconButton.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './IconButton.module.scss';
+
+const IconButton = (props) => {
+  const addClassname = () => {
```
Comment (FortizFil) [inline frontend/src/components/ui-kit/components/IconButton/IconButton.jsx:5]: Looks good, but optimize this

**Review by FortizFil — CHANGES_REQUESTED** [review]

**Review by FortizFil — APPROVED** [review]
Looks good

**Pull request merged**